scripts/help/cert-help-extraction.py

```diff
@@ -292,6 +292,12 @@ def helper(node):
                         full_name = repo_root.joinpath(rule_path, filename)
                         urllib.request.urlretrieve(url, full_name)
                         node['src'] = filename
+                # Replace check.svg and error.svg images with unicode characters
+                if node.name == 'img':
+                    if node['src'].endswith("check.svg"):
```

Comment:
just a quick check for including the check.svg and error.svg images in the repo, where did they come from, and did they have any attribution requirement?

Reply:
er sorry, so do we need the check.svg file checked into the repo then?